addons/l10n_in/models/chart_template.py

```diff
@@ -11,12 +11,13 @@ def _prepare_all_journals(self, acc_template_ref, company, journals_dict=None):
         res = super(AccountChartTemplate, self)._prepare_all_journals(acc_template_ref, company, journals_dict=journals_dict)
         if self == self.env.ref('l10n_in.indian_chart_template_standard'):
             for journal in res:
+                journal['l10n_in_gstin_partner_id'] = company.partner_id.id
```

Comment:
set value of l10n_in_gstin_partner_id for only sales and purchase journal

Reply:
Done